############################# import random
#import random
#x = random.randint(1,6)
#y = random.random()
#myList = ['rock', 'papers', 'scissors']
#z = random.choice(myList)

#cards = [1,2,3,4,5,6,7,8,9,10,"J","Q","K","A"]
#random.shuffle(cards)

#print(cards)

############################# exception = evento detectado durante uma execução interrompida do flow do programa
#numerator = int(input("Enter a number to divide: "))
#denominator = int(input("Enter a number to be divide by: "))
#result = numerator / denominator




#try:
    #numerator = int(input("Enter a number to divide: "))
    #denominator = int(input("Enter a number to be divide by: "))
    #result = numerator / denominator

#except ZeroDivisionError as e:
    #print(e)
    #print("You can't divide by zero idiot!")
#except ValueError as e:
    #print(e)
    #print("Enter only NUMBERS INTEGER plz, don't type letters :( ")
#except Exception as e:
    #print(e)
    #print("Something went wrong :( ")
#else:
   # print(result)
#finally:
    #print("This will always execute")
